frontend/src/v5/validation/containerAndFederationSchemes/validators.ts

```diff
@@ -58,21 +58,24 @@ export const name = trimmedString
 			defaultMessage: 'This name is already used within this project',
 		}),
 		(nameValue, testContext) => {
-			return !testContext.options.context.alreadyExistingNames.map((n) => n.trim().toLocaleLowerCase()).includes(nameValue?.toLocaleLowerCase());
+			if (!testContext.options?.context) return true;
+			return !testContext.options.context.alreadyExistingNames?.map((n) => n.trim().toLocaleLowerCase()).includes(nameValue?.toLocaleLowerCase());
 		},
 	);
 
 export const unit = Yup.string().required().oneOf(['mm', 'cm', 'dm', 'm', 'ft']).default('mm');
 export const type = Yup.string().required().default('Uncategorised');
 
+export const alphaNumericHyphens = /^[\w|_|-]*$/;
```

Comment:
alphanumeric is just one word. Also, not a big deal, but I have just found that regex entity `\w` already includes the underscore ("_")